src/App.tsx

```diff
@@ -17,6 +18,7 @@ function App() {
           <BrowserRouter>
             <AppRoutes />
           </BrowserRouter>
+          <Footer />
```

Comment:
In dashboard it will be overlapped by NavPanel. Maybe it's better to insert it separately for auth and base?